app/components/ui/Navigation.tsx

```diff
@@ -144,7 +136,7 @@ export const Navigation = () => {
               <div className={styles.navigationMenuTranslate}>
                 <GoogleTranslate />
               </div>
```

Comment:
I don't think it needs to be done here, or maybe not at all, but the nav section might load more cleanly if there's space allocated for translate before it loads, so users don't see the whole nav shift to the left once the translation dropdown loads.

Reply:
Added this in!